src/utilities/list_ROI_values.cxx

```diff
@@ -261,7 +261,8 @@ main(int argc, char *argv[])
 
 	for (int i=min_plane_number;i<=max_plane_number;i++)
 	  {
-	    out << std::setw(15) << *current_name_iter
+        out << std::setw(15) <<input_file
```

Comment:
white spaces seem a bit different. ideally don't use tab, but just spaces and make it align then (but it could be because of existing tabs)